clojure-ts-mode.el

```diff
@@ -746,33 +746,61 @@ The possible values for this variable are
      ((parent-is "list_lit") parent 1)
      ((parent-is "set_lit") parent 2))))
 
-(defvar clojure-ts--symbols-with-body-expressions-regexp
+(defvar clojure-ts--symbols-block-0-regexp
   (eval-and-compile
-    (rx (or
-         ;; Match def* symbols,
-         ;; we also explicitly do not match symbols beginning with
-         ;; "default" "deflate" and "defer", like cljfmt
-         (and line-start "def")
-         ;; Match with-* symbols
-         (and line-start "with-")
-         ;; Exact matches
-         (and line-start
-              (or "alt!" "alt!!" "are" "as->"
-                  "binding" "bound-fn"
-                  "case" "catch" "comment" "cond" "condp" "cond->" "cond->>"
-                  "delay" "do" "doseq" "dotimes" "doto"
-                  "extend" "extend-protocol" "extend-type"
-                  "fdef" "finally" "fn" "for" "future"
-                  "go" "go-loop"
-                  "if" "if-let" "if-not" "if-some"
-                  "let" "letfn" "locking" "loop"
-                  "match" "ns" "proxy" "reify" "struct-map"
-                  "testing" "thread" "try"
-                  "use-fixtures"
-                  "when" "when-first" "when-let" "when-not" "when-some" "while")
-              line-end))))
+    (rx (and line-start
+             (or "alt!" "alt!!"
+                 "comment"
+                 "cond"
+                 "delay"
+                 "do"
+                 "finally"
+                 "future"
+                 "go"
+                 "thread"
+                 "try"
+                 "with-out-str")
+             line-end)))
   "A regex to match symbols that are functions/macros with a body argument.
-Taken from cljfmt:
+Taken from cljfmt (all symbols with [[:block 0]] rule):
+https://github.com/weavejester/cljfmt/blob/fb26b22f569724b05c93eb2502592dfc2de898c3/cljfmt/resources/cljfmt/indents/clojure.clj")
```

Comment:
We might go with some version tag here instead, so the URL is a bit more informative.

Reply:
Done.